docs/SPEC.md

```diff
@@ -2757,6 +2829,56 @@ If you need to convert other columns use the `map` function directly with the `u
 
 [IMPL#242](https://github.com/influxdata/platform/issues/242) Update specification around type conversion functions.
 
+#### Errors
+
+Errors selects error tables from the meta channel and places them on the data channel.
+The input data channel is dropped.
+The output meta channel is empty.
+
+Example:
+
+    // Perform a simple downsample
+    from(bucket:"telegraf/autogen")
+        |> range(start: -5m)
+        |> aggregateWindow(every: 1m, fn: mean)
+        |> to(bucket: "telegraf/5m")
+        |> errors()
```

Comment:
I think it will be convenient to have a filter function as an optional parameter to errors(), or else make sure we name this nicely so that we can install a helper function that takes a function parameter.

Reply:
You could also just apply `filter()` to the output of `errors()`, since the errors will be represented just like regular data.